static/lang/re-en.json

```diff
@@ -2440,7 +2440,11 @@
                 "PiercingQuills": {
                     "Note": "Your quills unarmed attack deals @Localize[PF2E.PersistentDamage.Bleed1d4.success] on a critical hit."
                 },
-                "Quills": "Quills"
+                "Quills": "Quills",
+                "Spelunker": {
+                    "Climb": "If you roll a success on an Athletics check to Climb you get a critical success instead.",
+                    "SenseDirection": "If you roll a success on an Survival check to Sense Direction you get a critical success instead."
+                }
```

Comment:
These are generic enough that we should probably move them from under Shisk. I kind of want to migrate all these localizations for AdjustDegreeOfSuccess notes to be under one structure.

`PF2E.AdjustDegreeOfSuccess.Athletics.Climb.SuccessToCriticalSuccess` maybe. Should be large enough to justify another top level localization I think.